scripts/fleet-install-test.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+# Check whether insights-agent is part of the changed charts
+cd /charts
+CHANGED="$(ct list-changed --config ./scripts/ct.yaml)"
```

Comment:
We only run the fleet install test when there is a change in `insights-agent` chart, is that reasonable?

Reply:
Yes definitely desireable